security/cmd/chiron/main.go

```diff
@@ -167,22 +160,21 @@ func runWebhookController() {
 		os.Exit(1)
 	}
 
-	mutatingWebhookServiceNames := strings.Split(opts.mutatingWebhookServiceNames, ",")
-	validatingWebhookServiceNames := strings.Split(opts.validatingWebhookServiceNames, ",")
+	serviceNames := strings.Split(opts.serviceNames, ",")
+	serviceNamespaces := strings.Split(opts.serviceNamespaces, ",")
```

Comment:
Are these two slices of same length? If so need a validate

Reply:
The check for same length is in NewWebhookController.